google/cloud/bigquery/reservation_v1/services/reservation_service/client.py

```diff
@@ -147,17 +147,19 @@ def from_service_account_file(cls, filename: str, *args, **kwargs):
     from_service_account_json = from_service_account_file
 
     @staticmethod
-    def bi_reservation_path(project: str, location: str) -> str:
```

Comment:
@software-dov It looks like the method order may be unstable between generations